src/firefly/java/edu/caltech/ipac/firefly/server/query/lsst/LSSTQuery.java

```diff
@@ -421,4 +424,189 @@ else if ( classType.equalsIgnoreCase("NUMERIC") || classType.equalsIgnoreCase("D
 
     }
 
+    private static JSONObject jsonDataSet = LSSTQuery.getDataSet();
+    private static JSONObject getDataSet() {
+        JSONObject obj = new JSONObject();
+
+        // LSST dataset information for making table search request
+        String lsstDataSet = "{"
```

Comment:
I don't understand the purpose of this long json string. Is this a place holder? It is is it might be better to keep it in a file and access it as a resource from the jar